lib/foreman_tasks/tasks/cleanup.rake

```diff
@@ -9,6 +9,7 @@ namespace :foreman_tasks do
         * NOOP        : set to "true" if the task should not actuall perform the deletion
         * VERBOSE     : set to "true" for more verbose output
         * BATCH_SIZE  : the size of batches the tasks get processed in (1000 by default)
+        * TASK_BACKUP : the path where to place backup of cleaned tasks (set to empty to disable)
```

Comment:
Capturing the off-line conversation: we need to make the TASK_BACKUP field to be used when initializing the dynflow world to fully propagate to all areas. It should be true/false flag, rather than directory setup, to reduce the confusion around it